Assets/Scripts/KuriTextManager.cs

```diff
@@ -91,9 +119,17 @@ public enum PRIORITY {
                 audioSource.Play();
                 foreach (char letter in processTuple.text) {
                     kuriTextMesh.text += letter;
+                    //flickers light at each letter
```

Comment:
this is a great example of where you wouldn't need a comment. The function should be named for whatever it does. Thus that makes the comment redundant because you should be able to just know what it will do from the name. If you feel like you need a comment then you probably should just rename the function (the function name is decent imo but whatever you think is best)